client/src/partials/enterprises/enterprises.js

```diff
@@ -170,10 +170,10 @@ function EnterpriseController($translate, Enterprises, Currencies, Accounts, uti
   function deleteProject(id, pattern) {
     var params = {
       pattern: pattern,
-      elementName: $translate.instant('FORM.LABELS.PROJECT')
+      patternName: 'FORM.PATTERNS.PROJECT_NAME'
```

Comment:
I'm glad the `$translate` call has finally been moved into the modal.  This is a nice improvement.